content/Video_Display.md

```diff
@@ -566,8 +568,7 @@ coordinates of the top left corner of a window are WX-7,WY. The tiles
 for the window are stored in the Tile Data Table. Both the Background
 and the window share the same Tile Data Table.
 
-Both background and window can be disabled or enabled separately via
-bits in the LCDC register.
+Window can be disabled(0)/enabled(1) with 5th bits of LCDC register.
```

Comment:
I don't like the  *disabled(0)/enabled(1)*

Reply:
The grammatically correct form would be "The window can be disabled or enabled with bit 5 of the LCDC register." It's not the 5th bit (because we count from 0).